### PR TITLE
Add example for List subtraction

### PR DESCRIPTION
**Pull request by midnio**

### CODE REVIEW

**Comment by midnio** [comment]
Like this?

**Comment by gemantzu** [comment]
No, not really. On the same file you are editing (collections.md), on the top of the file, there is a version as well. this is the version you are supposed to edit each time you are doing some changes, not the one on the index.md of the language. Thanks!

File: en/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Elixir School
 layout: home
-version: 2.0.0
+version: 2.0.1
```
Comment (doomspork) [inline en/index.md]: @midnio can you undo this change?

File: en/lessons/basics/collections.md [hunk 2]
```diff
@@ -56,7 +56,12 @@ iex> [1,2,2,3,2,3] -- [1,2,3,2]
 [2, 3]
 ```
 
-**Note:** List subtraction uses [strict comparison](../basics/#comparison) to match the values.
+**Note:** List subtraction uses [strict comparison](../basics/#comparison) to match the values. For example:
```
Comment (doomspork) [inline en/lessons/basics/collections.md]: @midnio this is great 👍 Could we bump this file version from `1.2.3` to `1.2.4`?

**Comment by midnio** [comment]
@doomspork I finished.

**Comment by doomspork** [comment]
Thank you, thank you @midnio!
